frontend/src/components/ImplementationReportView/ImplementationReportView.tsx

```diff
@@ -0,0 +1,119 @@
+import { Card } from "react-bootstrap";
+import { useLoaderData, Link } from "react-router-dom";
+import { Table, Container } from "react-bootstrap";
+import { ImplementationMetadata } from "../../data/parseReportData";
+// @ts-ignore
+import LoadingAnimation from "../LoadingAnimation";
+import DialectCompliance from "./DialectCompliance";
+import { mapLanguage } from "../../data/mapLanguage";
+
+export const ImplementationReportView = () => {
+  const allImplementations = useLoaderData() as {
+    [key: string]: ImplementationMetadata;
+  };
+
+  const pathSegments = window.location.href.split("/");
```

Comment:
I haven't tried to fully understand what this does yet, so it's possible that this comment won't apply, but treating URLs as strings is a code smell. I believe we added a dependency on URI.js already, have a look and see whether it already does what you're doing manually here (which seems like maybe it's just pulling off the last path segment?).

Reply:
Oh yes, this path parameter should be retrieved from react router api, believe there is a hook for that